assets/app/styles/_components.less

```diff
@@ -152,9 +152,11 @@
 .pod-template-block {
   border-radius: 3px;
   margin-bottom: 10px;
+  margin-right: 10px;
   .pod-template {
     border: 1px dotted #ddd;
     border-radius: @border-radius-md;
+    max-width: 600px;
```

Comment:
I think always since there's a dotted border. It looks strange if it gets too wide.